holidays/locale/en_US/LC_MESSAGES/FI.po

```diff
@@ -86,3 +86,79 @@ msgstr "Christmas Day"
 #. Second Day of Christmas.
 msgid "Tapaninpäivä"
 msgstr "Second Day of Christmas"
+
+#. Runeberg Day
+msgid "Runebergin päivä"
+msgstr "Runeberg Day"
+
+#. Kalevala Day, Day of Finnish Culture
+msgid "Kalevalan päivä, suomalaisen kulttuurin päivä"
+msgstr "Kalevala Day, Day of Finnish Culture"
+
+#. Minna Canth Day, Day of Equality
+msgid "Minna Canthin päivä, tasa-arvon päivä"
+msgstr "Minna Canth Day, Day of Equality"
+
+#. Mikael Agricola Day, Day of the Finnish Language
+msgid "Mikael Agricolan päivä, suomen kielen päivä"
+msgstr "Mikael Agricola Day, Day of the Finnish Language"
+
+#. National War Veterans' Day
+msgid "Kansallinen veteraanipäivä"
+msgstr "National War Veterans' Day"
+
+#. Europe Day
+msgid "Eurooppa-päivä"
+msgstr "Europe Day"
+
+#. Mothers' Day
+msgid "Äitienpäivä"
+msgstr "Mothers' Day"
+
+#. J. V. Snellman Day, Day of Finnish Heritage
+msgid "J.V. Snellmanin päivä, suomalaisuuden päivä"
+msgstr "J. V. Snellman Day, Day of Finnish Heritage"
+
+#. Remembrance Day
+msgid "Kaatuneitten muistopäivä"
+msgstr "Remembrance Day"
+
+#. Flag Day of the Finnish Defence Forces
+msgid "Puolustusvoimain lippujuhlan päivä"
+msgstr "Flag Day of the Finnish Defence Forces"
+
+#. Eino Leino Day, Day of Summer and Poetry
+msgid "Eino Leinon päivä, runon ja suven päivä"
+msgstr "Eino Leino Day, Day of Summer and Poetry"
+
+#. Finland's Nature Day
+msgid "Suomen luonnon päivä"
+msgstr "Finland's Nature Day"
+
+#. Miina Sillanpää Day, Day of Civic Participation
+msgid "Miina Sillanpään ja kansalaisvaikuttamisen päivä"
+msgstr "Miina Sillanpää Day, Day of Civic Participation"
+
+#. Aleksis Kivi Day, Day of Finnish Literature
+msgid "Aleksis Kiven päivä, suomalaisen kirjallisuuden päivä"
+msgstr "Aleksis Kivi Day, Day of Finnish Literature"
+
+#. United Nations Day
+msgid "YK:n päivä"
+msgstr "United Nations Day"
+
+#. Finnish Swedish Heritage Day, svenska dagen
+msgid "Ruotsalaisuuden päivä, Kustaa Aadolfin päivä"
+msgstr "Finnish Swedish Heritage Day, svenska dagen"
```

Comment:
Not sure if "svenska dagen" translation is missing

Reply:
Seems that that's the official English version name according [to the Ministry of Interior](https://intermin.fi/en/flag-and-arms/flag-flying-days).
Also tbh I haven't really heard "Ruotsalaisuuden päivä" used among locals, everyone just refer to it as "Svenska dagen" (so literally "Swedish day" in Swedish 😅 ).